src/test/fuzz/transaction.cpp

```diff
@@ -111,4 +113,24 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         static const uint256 u256_max(uint256S("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"));
         TxToUniv(tx, u256_max, u);
     }
+
+    {
+        FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
+        const CScript script = ConsumeScript(fuzzed_data_provider);
```

Comment:
This code will never be executed when the buffer does not start with a valid transaction

Reply:
Oh, good catch! Thanks a lot for noticing! I missed the `return`:s above (L37/42/45/61). Now fixed. Please re-review updated version :)